src/csharp.tmLanguage.yml

```diff
@@ -3066,7 +3120,6 @@ repository:
       '7': { name: entity.name.variable.parameter.cs }
 
   type:
-    name: meta.type.cs
```

Comment:
By the way, although this is somewhat unrelated to the PR, I removed this line because `meta.` scope is meant to be for definition bodies.

`meta.type` is meant to be used for the class body starting from and including `public/internal class ... {` to the closing brace `}`.

Reply:
Sounds good. Thanks!